torch/nn/quantized/modules/linear.py

```diff
@@ -36,7 +36,8 @@ def forward(self, X):
 
     @staticmethod
     def from_float(mod):
-        return Quantize(mod.qparams[0].item(), mod.qparams[1].item(), torch.quint8)
+        qparams = mod.observer.calculate_qparams()
+        return Quantize(qparams[0].item(), qparams[1].item(), torch.quint8)
```

Comment:
shouldn't it be something like Observer.dtype? don't we want to make dtype configurable?